RestEqlCancellationIT fails in multijob-windows-compatibility builds

**Build scan**:

https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+multijob-windows-compatibility/os=windows-2016/43/console
[7.7.2] https://gradle-enterprise.elastic.co/s/e4zntze3jwrss
[7.8.0] https://gradle-enterprise.elastic.co/s/pszrdvqy6fuge
[7.9.0] https://gradle-enterprise.elastic.co/s/zewmmmpbi5pc2
https://gradle-enterprise.elastic.co/s/4ixn4nh5nn522

**Repro line**:
```
REPRODUCE WITH: gradlew ':x-pack:plugin:eql:internalClusterTest' --tests "org.elasticsearch.xpack.eql.action.RestEqlCancellationIT.testRestCancellation" \
  -Dtests.seed=A7224F6EA7FD550E \
  -Dtests.security.manager=true \
  -Dtests.locale=ar-QA \
  -Dtests.timezone=America/Cayenne \
  -Druntime.java=11
```

**Reproduces locally?**:
Doesn't reproduce on Linux nor Windows 10. Might be something server or CI specific.

**Applicable branches**:
master

**Failure history**:

So far it failed 6 times but only on multijob-windows-compatibility on os=windows-2016, os=windows=2019 and os=windows-2012-r2 builds.

**Failure excerpt**:

```
java.lang.AssertionError: 
Expected: null
     but: was <{
  "node" : "MVDL3H70QbGd9YymIbRqZQ",
  "id" : 155,
  "type" : "transport",
  "action" : "indices:data/read/eql",
  "start_time" : "2020-06-17T08:37:17.174Z",
  "start_time_in_millis" : 1592383037174,
  "running_time" : "10.3s",
  "running_time_in_nanos" : 10310585980,
  "cancellable" : true,
  "headers" : {
    "X-Opaque-Id" : "sWOaIWXLVO"
  }
}>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
	at org.junit.Assert.assertThat(Assert.java:956)
	at org.junit.Assert.assertThat(Assert.java:923)
	at org.elasticsearch.xpack.eql.action.RestEqlCancellationIT.lambda$testRestCancellation$0(RestEqlCancellationIT.java:133)
	at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:892)
	at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:865)
	at org.elasticsearch.xpack.eql.action.RestEqlCancellationIT.testRestCancellation(RestEqlCancellationIT.java:132)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at com.carrotsearch.randomizedtesting.RandomizedRunner.invoke(RandomizedRunner.java:1758)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$8.evaluate(RandomizedRunner.java:946)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$9.evaluate(RandomizedRunner.java:982)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$10.evaluate(RandomizedRunner.java:996)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at org.apache.lucene.util.TestRuleSetupTeardownChained$1.evaluate(TestRuleSetupTeardownChained.java:49)
	at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:45)
	at org.apache.lucene.util.TestRuleThreadAndTestName$1.evaluate(TestRuleThreadAndTestName.java:48)
	at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:64)
	at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:47)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:375)
	at com.carrotsearch.randomizedtesting.ThreadLeakControl.forkTimeoutingTask(ThreadLeakControl.java:824)
	at com.carrotsearch.randomizedtesting.ThreadLeakControl$3.evaluate(ThreadLeakControl.java:475)
	at com.carrotsearch.randomizedtesting.RandomizedRunner.runSingleTest(RandomizedRunner.java:955)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$5.evaluate(RandomizedRunner.java:840)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$6.evaluate(RandomizedRunner.java:891)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$7.evaluate(RandomizedRunner.java:902)
	at org.apache.lucene.util.AbstractBeforeAfterRule$1.evaluate(AbstractBeforeAfterRule.java:45)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at org.apache.lucene.util.TestRuleStoreClassName$1.evaluate(TestRuleStoreClassName.java:41)
	at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
	at com.carrotsearch.randomizedtesting.rules.NoShadowingOrOverridesOnMethodsRule$1.evaluate(NoShadowingOrOverridesOnMethodsRule.java:40)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at org.apache.lucene.util.TestRuleAssertionsRequired$1.evaluate(TestRuleAssertionsRequired.java:53)
	at org.apache.lucene.util.TestRuleMarkFailure$1.evaluate(TestRuleMarkFailure.java:47)
	at org.apache.lucene.util.TestRuleIgnoreAfterMaxFailures$1.evaluate(TestRuleIgnoreAfterMaxFailures.java:64)
	at org.apache.lucene.util.TestRuleIgnoreTestSuites$1.evaluate(TestRuleIgnoreTestSuites.java:54)
	at com.carrotsearch.randomizedtesting.rules.StatementAdapter.evaluate(StatementAdapter.java:36)
	at com.carrotsearch.randomizedtesting.ThreadLeakControl$StatementRunner.run(ThreadLeakControl.java:375)
	at com.carrotsearch.randomizedtesting.ThreadLeakControl.lambda$forkTimeoutingTask$0(ThreadLeakControl.java:831)
	at java.base/java.lang.Thread.run(Thread.java:834)
	Suppressed: java.lang.AssertionError: 
Expected: null
     but: was <{
  "node" : "MVDL3H70QbGd9YymIbRqZQ",
  "id" : 155,
  "type" : "transport",
  "action" : "indices:data/read/eql",
  "start_time" : "2020-06-17T08:37:17.174Z",
  "start_time_in_millis" : 1592383037174,
  "running_time" : "255.8ms",
  "running_time_in_nanos" : 255815180,
  "cancellable" : true,
  "headers" : {
    "X-Opaque-Id" : "sWOaIWXLVO"
  }
}>
		at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
		at org.junit.Assert.assertThat(Assert.java:956)
		at org.junit.Assert.assertThat(Assert.java:923)
		at org.elasticsearch.xpack.eql.action.RestEqlCancellationIT.lambda$testRestCancellation$0(RestEqlCancellationIT.java:133)
		at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:880)
		... 39 more
	Suppressed: java.lang.AssertionError: 
Expected: null
     but: was <{
  "node" : "MVDL3H70QbGd9YymIbRqZQ",
  "id" : 155,
  "type" : "transport",
  "action" : "indices:data/read/eql",
  "start_time" : "2020-06-17T08:37:17.174Z",
  "start_time_in_millis" : 1592383037174,
  "running_time" : "293.8ms",
  "running_time_in_nanos" : 293812314,
  "cancellable" : true,
  "headers" : {
    "X-Opaque-Id" : "sWOaIWXLVO"
  }
}>
		at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
		at org.junit.Assert.assertThat(Assert.java:956)
		at org.junit.Assert.assertThat(Assert.java:923)
		at org.elasticsearch.xpack.eql.action.RestEqlCancellationIT.lambda$testRestCancellation$0(RestEqlCancellationIT.java:133)
		at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:880)
		... 39 more
	Suppressed: java.lang.AssertionError: 
Expected: null
     but: was <{
  "node" : "MVDL3H70QbGd9YymIbRqZQ",
  "id" : 155,
  "type" : "transport",
  "action" : "indices:data/read/eql",
  "start_time" : "2020-06-17T08:37:17.174Z",
  "start_time_in_millis" : 1592383037174,
  "running_time" : "296.7ms",
  "running_time_in_nanos" : 296749664,
  "cancellable" : true,
  "headers" : {
    "X-Opaque-Id" : "sWOaIWXLVO"
  }
}>
		at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
		at org.junit.Assert.assertThat(Assert.java:956)
		at org.junit.Assert.assertThat(Assert.java:923)
		at org.elasticsearch.xpack.eql.action.RestEqlCancellationIT.lambda$testRestCancellation$0(RestEqlCancellationIT.java:133)
		at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:880)
		... 39 more
	Suppressed: java.lang.AssertionError: 
Expected: null
     but: was <{
  "node" : "MVDL3H70QbGd9YymIbRqZQ",
  "id" : 155,
  "type" : "transport",
  "action" : "indices:data/read/eql",
  "start_time" : "2020-06-17T08:37:17.174Z",
  "start_time_in_millis" : 1592383037174,
  "running_time" : "301.5ms",
  "running_time_in_nanos" : 301598313,
  "cancellable" : true,
  "headers" : {
    "X-Opaque-Id" : "sWOaIWXLVO"
  }
}>
		at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
		at org.junit.Assert.assertThat(Assert.java:956)
		at org.junit.Assert.assertThat(Assert.java:923)
		at org.elasticsearch.xpack.eql.action.RestEqlCancellationIT.lambda$testRestCancellation$0(RestEqlCancellationIT.java:133)
		at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:880)
		... 39 more
	Suppressed: java.lang.AssertionError: 
Expected: null
     but: was <{
  "node" : "MVDL3H70QbGd9YymIbRqZQ",
  "id" : 155,
  "type" : "transport",
  "action" : "indices:data/read/eql",
  "start_time" : "2020-06-17T08:37:17.174Z",
  "start_time_in_millis" : 1592383037174,
  "running_time" : "310.4ms",
  "running_time_in_nanos" : 310496892,
  "cancellable" : true,
  "headers" : {
    "X-Opaque-Id" : "sWOaIWXLVO"
  }
}>
		at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
		at org.junit.Assert.assertThat(Assert.java:956)
		at org.junit.Assert.assertThat(Assert.java:923)
		at org.elasticsearch.xpack.eql.action.RestEqlCancellationIT.lambda$testRestCancellation$0(RestEqlCancellationIT.java:133)
		at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:880)
		... 39 more
	Suppressed: java.lang.AssertionError: 
Expected: null
     but: was <{
  "node" : "MVDL3H70QbGd9YymIbRqZQ",
  "id" : 155,
  "type" : "transport",
  "action" : "indices:data/read/eql",
  "start_time" : "2020-06-17T08:37:17.174Z",
  "start_time_in_millis" : 1592383037174,
  "running_time" : "328ms",
  "running_time_in_nanos" : 328077472,
  "cancellable" : true,
  "headers" : {
    "X-Opaque-Id" : "sWOaIWXLVO"
  }
}>
		at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
		at org.junit.Assert.assertThat(Assert.java:956)
		at org.junit.Assert.assertThat(Assert.java:923)
		at org.elasticsearch.xpack.eql.action.RestEqlCancellationIT.lambda$testRestCancellation$0(RestEqlCancellationIT.java:133)
		at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:880)
		... 39 more
	Suppressed: java.lang.AssertionError: 
Expected: null
     but: was <{
  "node" : "MVDL3H70QbGd9YymIbRqZQ",
  "id" : 155,
  "type" : "transport",
  "action" : "indices:data/read/eql",
  "start_time" : "2020-06-17T08:37:17.174Z",
  "start_time_in_millis" : 1592383037174,
  "running_time" : "361.2ms",
  "running_time_in_nanos" : 361274084,
  "cancellable" : true,
  "headers" : {
    "X-Opaque-Id" : "sWOaIWXLVO"
  }
}>
		at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
		at org.junit.Assert.assertThat(Assert.java:956)
		at org.junit.Assert.assertThat(Assert.java:923)
		at org.elasticsearch.xpack.eql.action.RestEqlCancellationIT.lambda$testRestCancellation$0(RestEqlCancellationIT.java:133)
		at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:880)
		... 39 more
	Suppressed: java.lang.AssertionError: 
Expected: null
     but: was <{
  "node" : "MVDL3H70QbGd9YymIbRqZQ",
  "id" : 155,
  "type" : "transport",
  "action" : "indices:data/read/eql",
  "start_time" : "2020-06-17T08:37:17.174Z",
  "start_time_in_millis" : 1592383037174,
  "running_time" : "426.7ms",
  "running_time_in_nanos" : 426730858,
  "cancellable" : true,
  "headers" : {
    "X-Opaque-Id" : "sWOaIWXLVO"
  }
}>
		at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
		at org.junit.Assert.assertThat(Assert.java:956)
		at org.junit.Assert.assertThat(Assert.java:923)
		at org.elasticsearch.xpack.eql.action.RestEqlCancellationIT.lambda$testRestCancellation$0(RestEqlCancellationIT.java:133)
		at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:880)
		... 39 more
	Suppressed: java.lang.AssertionError: 
Expected: null
     but: was <{
  "node" : "MVDL3H70QbGd9YymIbRqZQ",
  "id" : 155,
  "type" : "transport",
  "action" : "indices:data/read/eql",
  "start_time" : "2020-06-17T08:37:17.174Z",
  "start_time_in_millis" : 1592383037174,
  "running_time" : "556.6ms",
  "running_time_in_nanos" : 556607604,
  "cancellable" : true,
  "headers" : {
    "X-Opaque-Id" : "sWOaIWXLVO"
  }
}>
		at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
		at org.junit.Assert.assertThat(Assert.java:956)
		at org.junit.Assert.assertThat(Assert.java:923)
		at org.elasticsearch.xpack.eql.action.RestEqlCancellationIT.lambda$testRestCancellation$0(RestEqlCancellationIT.java:133)
		at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:880)
		... 39 more
	Suppressed: java.lang.AssertionError: 
Expected: null
     but: was <{
  "node" : "MVDL3H70QbGd9YymIbRqZQ",
  "id" : 155,
  "type" : "transport",
  "action" : "indices:data/read/eql",
  "start_time" : "2020-06-17T08:37:17.174Z",
  "start_time_in_millis" : 1592383037174,
  "running_time" : "814.4ms",
  "running_time_in_nanos" : 814455940,
  "cancellable" : true,
  "headers" : {
    "X-Opaque-Id" : "sWOaIWXLVO"
  }
}>
		at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
		at org.junit.Assert.assertThat(Assert.java:956)
		at org.junit.Assert.assertThat(Assert.java:923)
		at org.elasticsearch.xpack.eql.action.RestEqlCancellationIT.lambda$testRestCancellation$0(RestEqlCancellationIT.java:133)
		at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:880)
		... 39 more
	Suppressed: java.lang.AssertionError: 
Expected: null
     but: was <{
  "node" : "MVDL3H70QbGd9YymIbRqZQ",
  "id" : 155,
  "type" : "transport",
  "action" : "indices:data/read/eql",
  "start_time" : "2020-06-17T08:37:17.174Z",
  "start_time_in_millis" : 1592383037174,
  "running_time" : "1.3s",
  "running_time_in_nanos" : 1328060576,
  "cancellable" : true,
  "headers" : {
    "X-Opaque-Id" : "sWOaIWXLVO"
  }
}>
		at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
		at org.junit.Assert.assertThat(Assert.java:956)
		at org.junit.Assert.assertThat(Assert.java:923)
		at org.elasticsearch.xpack.eql.action.RestEqlCancellationIT.lambda$testRestCancellation$0(RestEqlCancellationIT.java:133)
		at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:880)
		... 39 more
	Suppressed: java.lang.AssertionError: 
Expected: null
     but: was <{
  "node" : "MVDL3H70QbGd9YymIbRqZQ",
  "id" : 155,
  "type" : "transport",
  "action" : "indices:data/read/eql",
  "start_time" : "2020-06-17T08:37:17.174Z",
  "start_time_in_millis" : 1592383037174,
  "running_time" : "2.3s",
  "running_time_in_nanos" : 2353503444,
  "cancellable" : true,
  "headers" : {
    "X-Opaque-Id" : "sWOaIWXLVO"
  }
}>
		at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
		at org.junit.Assert.assertThat(Assert.java:956)
		at org.junit.Assert.assertThat(Assert.java:923)
		at org.elasticsearch.xpack.eql.action.RestEqlCancellationIT.lambda$testRestCancellation$0(RestEqlCancellationIT.java:133)
		at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:880)
		... 39 more
	Suppressed: java.lang.AssertionError: 
Expected: null
     but: was <{
  "node" : "MVDL3H70QbGd9YymIbRqZQ",
  "id" : 155,
  "type" : "transport",
  "action" : "indices:data/read/eql",
  "start_time" : "2020-06-17T08:37:17.174Z",
  "start_time_in_millis" : 1592383037174,
  "running_time" : "4.4s",
  "running_time_in_nanos" : 4403399482,
  "cancellable" : true,
  "headers" : {
    "X-Opaque-Id" : "sWOaIWXLVO"
  }
}>
		at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:18)
		at org.junit.Assert.assertThat(Assert.java:956)
		at org.junit.Assert.assertThat(Assert.java:923)
		at org.elasticsearch.xpack.eql.action.RestEqlCancellationIT.lambda$testRestCancellation$0(RestEqlCancellationIT.java:133)
		at org.elasticsearch.test.ESTestCase.assertBusy(ESTestCase.java:880)
		... 39 more
```

--- a/x-pack/plugin/eql/src/internalClusterTest/java/org/elasticsearch/xpack/eql/action/RestEqlCancellationIT.java
+++ b/x-pack/plugin/eql/src/internalClusterTest/java/org/elasticsearch/xpack/eql/action/RestEqlCancellationIT.java
@@ -18,7 +18,6 @@ import org.elasticsearch.common.settings.Settings;
 import org.elasticsearch.plugins.Plugin;
 import org.elasticsearch.tasks.Task;
 import org.elasticsearch.tasks.TaskInfo;
-import org.elasticsearch.test.junit.annotations.TestIssueLogging;
 import org.elasticsearch.transport.Netty4Plugin;
 import org.elasticsearch.transport.TransportService;
 import org.elasticsearch.transport.nio.NioTransportPlugin;
@@ -38,8 +37,6 @@ import static org.hamcrest.Matchers.instanceOf;
 import static org.hamcrest.Matchers.notNullValue;
 import static org.hamcrest.Matchers.nullValue;
 
-@TestIssueLogging(value = "org.elasticsearch.xpack.eql.action:TRACE",
-    issueUrl = "https://github.com/elastic/elasticsearch/issues/58270")
 public class RestEqlCancellationIT extends AbstractEqlBlockingIntegTestCase {
 
     private static String nodeHttpTypeKey;

